src/components/login/LoginForm.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './LoginForm.css';
+import {Link} from 'react-router-dom';
+
+function LoginForm({email, password, onSubmit, handleInputChange}) {
+  return (
+    <div className='container-fluid'>
+      <div className='row vertical-center'>
+        <div
+          className='card card-nav-tabs text-center p-3 mx-auto col-lg-8 col-md-10 col-sm-12'
+          data-color='orange'>
+          <div className='card-header card-header-warning'>
+            <div className='logo'>
+              <Link to='/'>
+                <img
+                  src='https://www.lesewert.de/files/lesewert/img/lesewert_logo.svg'
+                  alt='Lesewert'
+                />
+              </Link>
+            </div>
+            <br />
+            <p>Welcome to Lesewert Dashboard!</p>
+          </div>
+          <form onSubmit={onSubmit}>
+            <div className='form-group mt-5'>
+              <label htmlFor='exampleInputEmail1'>Email address</label>
+              <input
+                type='email'
+                name='email'
+                value={email}
+                onChange={handleInputChange}
+                className='form-control'
+                id='exampleInputEmail1'
+                aria-describedby='emailHelp'
+                placeholder='Enter email'
+                required
+              />
+            </div>
+            <div className='form-group'>
+              <label htmlFor='exampleInputPassword1'>Password</label>
+              <input
+                type='password'
+                name='password'
+                value={password}
+                onChange={handleInputChange}
+                className='form-control'
+                id='exampleInputPassword1'
+                placeholder='Password'
+                required
+              />
+            </div>
+            {/* <div className='form-check'>
```

Comment:
remove comments

Reply:
I kept this comment to use it in the future but I can delete it if we are not gonna use it